setup.py

```diff
@@ -13,7 +13,7 @@
 else:
     # py2
     requirements.append("enum34")
-    requirements.append("vivisect")
+    requirements.append("vivisect @ https://github.com/williballenthin/vivisect/tarball/master#egg=vivisect")
```

Comment:
whoa, this is magic!

Reply:
we should probably pin this to a particular tag, given that viv has breaking changes occasionally.